epicure-frontend/src/components/Hero/Hero.tsx

```diff
@@ -0,0 +1,27 @@
+import './Hero.scss';
+import SearchIcon from '../../assets/images/search-icon.svg'
+
+const Hero = () => {
+    return (
+        <>
+            <div className='hero-container' >
+                <div className='hero-overlay'>
+                    <div className='hero-content'>
+                        <div className='hero-text'>Epicure works with the top chef restaurants in Tel Aviv</div>
+                        <div className='search-bar'>
```

Comment:
set as input and not div

Reply:
I have in this div a button and an input, if I will change this the input wont be able to contain the button.